specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json

```diff
@@ -0,0 +1,723 @@
+              "$ref": "#/definitions/LongTermRetentionPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidParameterValue - An invalid value was given to a parameter.\n\n * 400 LongTermRetentionPolicyNotSupported - Long Term Retention is not supported on this database.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create or update the long term retention policy for the database.": {
+            "$ref": "./examples/LongTermRetentionPolicyCreateOrUpdate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/longTermRetentionPolicies": {
+      "get": {
+        "tags": [
+          "LongTermRetentionPolicies"
+        ],
+        "description": "Gets a database's long term retention policy.",
+        "operationId": "LongTermRetentionPolicies_ListByDatabase",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully get the policy.",
+            "schema": {
+              "$ref": "#/definitions/LongTermRetentionPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Get the long term retention policy for the database.": {
+            "$ref": "./examples/LongTermRetentionPolicyGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LongTermRetentionBackupProperties": {
+      "description": "Properties of a long term retention backup",
+      "type": "object",
+      "properties": {
+        "serverName": {
+          "description": "The server name that the backup database belong to.",
+          "type": "string",
+          "readOnly": true
+        },
+        "serverCreateTime": {
+          "format": "date-time",
+          "description": "The create time of the server.",
+          "type": "string",
+          "readOnly": true
+        },
+        "databaseName": {
```

Comment:
same as above - fully qualified arm resource id.

Reply:
Same issue - database might be deleted

Reply:
got it